### ISSUE TITLE
en.wikipedia.org blocked on my private wiki

### ISSUE THREAD

**Issue by nyuszika7h** [EFForg/privacybadgerfirefox#502]
It makes a request to Wikipedia to load wikEd. For some unknown reason, Privacy Badger is blocking this, and worse, it won't let me change that – the switch changes back to red after reload.

**Comment by cooperq** [comment]
Which domain exactly is being blocked. Also the fact that it is moving back to red seems like an even more disturbing bug.

**Comment by nyuszika7h** [comment]
Like the title says, en.wikipedia.org.

**Comment by nyuszika7h** [comment]
This may be relevant, though it's not blocking Wikipedia in this particular instance:

```
Found hi-entropy localStorage:  86.09645967306555  bits "http://flarar.cadoth.net/wiki/Main_Page" wikEdDisabled
```

**Comment by cooperq** [comment]
I think that this was due to the other issues we saw where privacy badger was overzealously blocking things. It should be fixed as of 1.0.3 - For now you can just manually unblock en.wikipedia.org of course.
